src/Auth/Auth.php

```diff
@@ -81,7 +81,7 @@ public static function login( array $input ): array {
 		// Set the auth cookie if the provider is configured to use it.
 		$config = $client->get_config();
 		if ( ! empty( $config['loginOptions']['useAuthenticationCookie'] ) ) {
-			wp_set_auth_cookie( $user->ID, false );
+			AuthCookie::custom_wp_auth_cookie( $user->ID, false );
```

Comment:
Issue is you called the method `::wp_custom_auth_cookie()`. my recommendation below to rename the function to `::set_auth_cookie()` will help with this sort of confusion.

Reply:
Ps, running `composer phpstan` should help catch these issues.

Reply:
Renamed to ::set_auth_cookie()